### DISCUSSION TITLE
Import assets: Override default compression method for high-resolution images

### DISCUSSION THREAD

**Discussion by ShlomiRex** [godotengine/godot-proposals#8476] [Engine Core]
Related to: https://github.com/godotengine/godot/issues/85074
My proposal is to override the default compression method for importing images to `COMPRESS_VRAM_COMPRESSED` instead of `COMPRESS_LOSSLESS` (the default method) **for large textures**.

TL;DR when importing an FBX model (57MB in this example), we don't use any compression, and it takes 2 minutes and 17 seconds to load a single FBX model.
After investigating, I analyzed the import time for the same model but with `COMPRESS_VRAM_COMPRESSED` compression and it takes 28 seconds instead.

That is 87% decrease in the time to import the FBX model.

We should only override the default compression method if we detect that the texture is large. Now this is up to debate, what does it mean when the texture is large is open to interpretation.

But in this case, the texture is 5MB with resolution 8192x8192.
In my opinion we should override the compression method for large textures, just so the user won't get locked for a long time. And if he wants to use lossless compression, he can choose in the Import dock to re-import with lossless compression.

Note: it is also related to this video, where Godot is compared to Unity: https://www.youtube.com/watch?v=wZFcTF718To

**Comment by Calinou** [comment]
This is likely a bug: https://github.com/godotengine/godot/issues/85074#issuecomment-1818914806
